lib/mix/tasks/phx.gen.ex

```diff
@@ -18,6 +18,33 @@ defmodule Mix.Tasks.Phx.Gen do
   | `phx.gen.live`     | x | x | x |   |   | x |
   | `phx.gen.json`     | x | x | x | x | x |   |
   | `phx.gen.html`     | x | x | x | x | x |   |
+
+  ## Customizing generators
+  
+  You can override the default templates used by generators.
+  
+  For example, to customize `phx.gen.live`, you can copy and edit the generator templates.
+  A common pattern is to place them under `priv/templates/phx.gen.live/` in your project:
+  
+  Create the directory for your custom `phx.gen.live` templates:
+
+  ```console
+  $ mkdir -p priv/templates/phx.gen.live
+  ```
+
+  Copy the default phx.gen.live generator templates into your project so you can customize them:
+
+  ```console
+  $ cp -r deps/phoenix/priv/templates/phx.gen.live/* priv/templates/phx.gen.live/
+  ```
```

Comment:
```suggestion
  For example, to customize `phx.gen.live`, you can copy and edit the generator templates
  to your own project's priv folder:
  
  First, create the directory for your custom `phx.gen.live` templates:

  ```console
  $ mkdir -p priv/templates/phx.gen.live
  ```

  Next, copy the default phx.gen.live generator templates into your project so you can customize them:

  ```console
  $ cp -r deps/phoenix/priv/templates/phx.gen.live/* priv/templates/phx.gen.live/
  ```
```